src/js/records/requests/asset-create.record.js

```diff
@@ -1,11 +1,17 @@
 import { ASSET_POLICIES } from '@tokend/js-sdk'
+
 import { RequestRecord } from '../request-record'
+import { AssetRecord } from '../entities/asset.record'
+
 import _get from 'lodash/get'
 
 export class AssetCreateRequestRecord extends RequestRecord {
   constructor (record, details = {}) {
     super(record)
 
+    this.asset = new AssetRecord(_get(
+      this._record, 'details.assetCreate'
```

Comment:
are you sure the JSON you get inside `details.assetCreate` is similar to the JSON you get when requesting the `/assets`? Even though it is, the details of the request and the  asset itself are completely different resources, and their similarity is just a lucky circumstances

Reply:
https://github.com/tokend/web-client/pull/37#discussion_r254729398